_posts/2024-10-15-Announcing-Data-Prepper-2.10.0.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: Announcing Data Prepper 2.10.0
+authors:
+- dvenable
+- TODO
+date: 2024-10-15 12:30:00 -0600
+categories:
+  - releases
+excerpt: Data Prepper 2.10.0 offers an OpenSearch _bulk API and reads from Amazon Kinesis
+meta_keywords: Data Prepper, _bulk, Amazon Kinesis Data Streams, OTel Logs, OTLP JSON
+meta_description: Data Prepper 2.10.0 offers a source that simulates the OpenSearch _bulk API and another source for reading from Amazon Kinesis Data Streams.
```

Comment:
Updated meta: 

_**meta_keywords:**_  Data Prepper, OpenSearch bulk API, Kinesis data streams, Kafka, SASL/SCRAM authentication, streaming data ingestion

_**meta_description:**_ Data Prepper 2.10.0 adds OpenSearch API and Kinesis Data Streams sources for seamless ingestion, plus Kafka SASL/SCRAM support and OpenTelemetry log parsing.